### DISCUSSION TITLE
Navigation from Create/Edit form view to List view

### DISCUSSION THREAD

**Discussion by Adityacs001** [keystonejs/keystone#7557] [Questions] [answered]
Current redirection from create/edit action is on detail view. How we can redirect user to list view instead of detail view.

User have to click multiple to create new item to list.

Is it possible to override redirection of save action  to custom view.

**Comment by raveling** [comment, marked as answer]
@Adityacs001 There's no easy way to achieve this right now. We're thinking about ways to make it easier for users to create new items with less clicks.

**Reply by Adityacs001** [reply]
Thanks @raveling for response.
